site/source/tutorials/001_static_content/index.md

```diff
@@ -77,7 +77,7 @@ Now, let's look at the `v` function again. We are intentionally avoiding somethi
 
 In traditional web applications, keeping the DOM and JavaScript application logic in sync led to significant complexity and inefficiency for non-trivial applications. When building applications with numerous changes to state and data, the virtual DOM approach can greatly simplify your application logic and improve performance. A virtual DOM serves as an intermediary between your application logic and what is rendered in the real DOM on the page.
 
-Dojo 2 leverages the [Maquette](http://maquettejs.org/) virtual DOM library to determine the most efficient way to interact with the DOM elements in your view. An additional benefit of the virtual DOM is that it facilitates a reactive programming style which simplifies your application. To learn more about the virtual DOM or reactive programming in Dojo 2, check out the [Working with a Virtual DOM](../comingsoon.html) and [Reactive Programming](../comingsoon.html) articles in the reference section. For now, let's get back to our application and make some more changes.
+Dojo 2 leverages its own virtual DOM library, initially created as a fork from [Maquette](http://maquettejs.org/), to determine the most efficient way to interact with the DOM elements in your view. An additional benefit of the virtual DOM is that it facilitates a reactive programming style which simplifies your application. To learn more about the virtual DOM or reactive programming in Dojo 2, check out the [Working with a Virtual DOM](../comingsoon.html) and [Reactive Programming](../comingsoon.html) articles in the reference section. For now, let's get back to our application and make some more changes.
```

Comment:
https://dojo.io/docs/fundamentals/working_with_virtual_dom/index.html and https://dojo.io/docs/fundamentals/reactive_programming/index.html exist now, so we should link to those rather than coming soon